frontend/packages/ceph-storage-plugin/src/components/modals/storage-pool-modal/storage-pool-modal.tsx

```diff
@@ -200,6 +221,7 @@ export const StoragePoolModal = withHandlePromise((props: StoragePoolModalProps)
         replicated: {
           size: Number(replicaSize),
         },
+        failureDomain,
```

Comment:
it is failureDomain: failureDomain, looks like linit  was modified to failureDomain.  

I think keyName and value var name is the same we don't need to mention like failureDomain: failureDomain